Maryia_Bychkova_Application/Program.cs

```diff
@@ -100,4 +101,178 @@ public void Task_7()
 
     }
 
-}
\ No newline at end of file
+}
+
+
+class Homework_2
+{
+    public void Task_1()
+    {
+        Console.WriteLine("Enter the number:");
+        var numberToString = Console.ReadLine();
+        var numberToInt = Convert.ToInt32(numberToString);
+        Console.WriteLine(numberToInt);
+
+        Console.WriteLine("Enter the number:");
+        var numberToDouble = Console.ReadLine();
+        var number1ToInt = Convert.ToInt32(numberToDouble);
+        Console.WriteLine(number1ToInt);
+
+        Console.WriteLine("Enter the number:");
+        var numberToByte = Console.ReadLine();
+        var number2ToInt = Convert.ToInt32(numberToByte);
+        Console.WriteLine(number2ToInt);
+
+        Console.WriteLine("Enter the number:");
+        var numberToChar = Console.ReadLine();
+        var number3ToInt = Convert.ToInt32(numberToChar);
+        Console.WriteLine(number3ToInt);
+
+    }
+    public void Task_2()
+    {
+        Console.WriteLine("Enter the number:"); 
+        var numberToString = Console.ReadLine();
+        var numberToint = int.Parse(numberToString);
+        Console.WriteLine(numberToint);
+
+        Console.WriteLine("Enter the number:");
+        var numberToDouble = Console.ReadLine();
+        var number1Toint = int.Parse(numberToDouble);
```

Comment:
and here instead  of int.Parse use double.Parse

Reply:
+